crates/ruff/resources/test/fixtures/pylint/sys_exit_alias_3.py

```diff
@@ -7,3 +7,10 @@
 def main():
     exit(1)
     quit(1)
+
+
+def main():
+    exit = 1
+
+    exit(1)
+    quit(1)
```

Comment:
The existing version of `resolve_binding` in `sys_exit_alias` didn't take into account that bindings might be overridden. We now avoid trying to fix this (since the `exit` in `from sys import exit` (out-of-view, on Line 1) is overridden here).